### ISSUE TITLE
Build fails on RHEL 6.5 after v8.1.1147

### ISSUE THREAD

**Issue by puremourning** [vim/vim#4234]
**Describe the bug**

Build fails due to `xgettext`/`msgfmt` incompatibility in RHEL 6.5 after patch 8.1.1147.

**To Reproduce**

```
$  git checkout v8.1.1147^
$ make distclean
$ ./configure ...
$ make
```

succeeds, but:

```
git checkout v8.1.1147
$ make distclean
$ ./configure ...
$ make
```
fails with:

```
make[2]: Entering directory `/mma/users/benj/proj/vim/vim.git/src/po'
cd ..; OLD_PO_FILE_INPUT=yes OLD_PO_FILE_OUTPUT=yes xgettext --default-domain=vim \
                --add-comments --keyword=_ --keyword=N_ --keyword=NGETTEXT:1,2 \
                *.c if_perl.xs GvimExt/gvimext.cpp globals.h if_py_both.h vim.h \
                po/gvim.desktop.in po/vim.desktop.in
gcc -std=gnu99 -c -I. -Iproto -DHAVE_CONFIG_H -DFEAT_GUI_GTK  -pthread -I/usr/include/gtk-2.0 -I/usr/lib64/gtk-2.0/include -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng12     -g -O2 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1        -o objects/pathdef.o auto/pathdef.c
gcc -std=gnu99 -c -I. -Iproto -DHAVE_CONFIG_H -DFEAT_GUI_GTK  -pthread -I/usr/include/gtk-2.0 -I/usr/lib64/gtk-2.0/include -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng12     -g -O2 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1        version.c -o objects/version.o
xgettext: warning: file `if_perl.xs' extension `xs' is unknown; will try C
xgettext: warning: file `po/gvim.desktop.in' extension `desktop' is unknown; will try C
xgettext: warning: file `po/vim.desktop.in' extension `desktop' is unknown; will try C
mv -f ../vim.po vim.pot
msgfmt --desktop -d . --template gvim.desktop.in -o gvim.desktop
msgfmt --desktop -d . --template vim.desktop.in -o vim.desktop
msgfmt: unrecognized option '--desktop'
Try `msgfmt --help' for more information.
make[2]: *** [gvim.desktop] Error 1
make[2]: *** Waiting for unfinished jobs....
msgfmt: unrecognized option '--desktop'
Try `msgfmt --help' for more information.
make[2]: *** [vim.desktop] Error 1
make[2]: Leaving directory `/mma/users/benj/proj/vim/vim.git/src/po'
make[1]: *** [languages] Error 2
make[1]: *** Waiting for unfinished jobs....
````

**Expected behavior**

Build succeeds on this platform. Let me know if the build time requirement is changed.

**Environment (please complete the following information):**
 - 8.1.1147+
 - OS: RHEL 6.5
 - Terminal: Xterm

**Additional context**

* Build succeeds with `--disable-nls`.
* xgettext version:

```
$ xgettext --version
xgettext (GNU gettext-tools) 0.17
Copyright (C) 1995-1998, 2000-2007 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Written by Ulrich Drepper.
```

**Comment by puremourning** [comment]
[config.log](https://gist.github.com/puremourning/a598d8ff18d3a646ab1d7057c954d6cb)

**Comment by chrisbra** [comment]
Hm, can you install a newer gettext version? Patch 8.1.1147 requires a gettext version 0.19.8

**Comment by chrisbra** [comment]
see also https://groups.google.com/forum/#!topic/vim_dev/6KVuMpzCTpI

**Comment by chrisbra** [comment]
I Bram already fixed it: 26096cc96fc402db45948b3b662a380281585649

Sorry for causing this.

**Comment by puremourning** [comment]
Thanks for the rapid response all!
